### DISCUSSION TITLE
Preloader for Galley

### DISCUSSION THREAD

**Discussion by Soeqnl** [dimsemenov/PhotoSwipe#2093] [Q&A]
Dear,

We've been using Photoswipe with great pleasure. However, recently, our photo albums have been experiencing freezing issues due to the extended loading time caused by loading all photos at once. You can find an example of this issue here: https://soeq.nl/q/eindejaarsshow-piushaven-tilburg-2023-2024/

I've been searching for a solution regarding the preload mechanism, also known as lazy loading, but I'm unsure about the specific adjustments needed in my photoswipe.js file. Since Photoswipe is integrated into the theme, I'm uncertain about the best approach to tackle this issue.

Best regards,
Gijs Franken
soeq.nl

**Comment by dimsemenov** [comment]
PhotoSwipe uses `preload` option to control the number of images to lazy-load https://photoswipe.com/options/#preload

You should probably ask your theme developer where it can be modified.
